README.md

```diff
@@ -86,35 +76,33 @@ Extends [`CompileOptions`](https://mdxjs.com/packages/mdx/#compileoptions)
 
 ###### Additional Properties
 
-- `lazyImages` (`boolean`, optional) -- Load images lazily.
-- `safeMode` (`boolean`, optional) -- Extract script tags from `HTMLBlock`s
-- `components` (`Record<string, string>`, optional) -- An object of tag names to mdx.
-- `copyButtons` (`Boolean`, optional) — Automatically insert a button to copy a block of text to the clipboard. Currently used on `<code>` elements.
+- `lazyImages` (`boolean`, optional) -- load images lazily
+- `safeMode` (`boolean`, optional) -- extract script tags from `HTMLBlock`s
+- `components` (`Record<string, string>`, optional) -- an object of tag names to mdx.
+- `copyButtons` (`Boolean`, optional) — add a Copy button to code blocks
 
 ### `RunOpts`
 
 Extends [`RunOptions`](https://mdxjs.com/packages/mdx/#runoptions)
 
 ###### Additional Properties
 
-- `components` (`Record<string, MDXModule>`, optional) -- An object of tag names to executed components.
-- `imports` (`Record<string, unknown>`, optional) -- An object of modules to import.
+- `components` (`Record<string, MDXModule>`, optional) -- an object of tag names to executed components
```

Comment:
@rafegoldberg Is this supposed to be 'execute' or 'executed'?

Reply:
T'was intentional, I think? But I updated the language to (hopefully) clarify!